.github/ISSUE_TEMPLATE/launch-analytics.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Launch - Analytics
+about: Prepare for a site launch with a checklist of common steps.
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+Please check off line-items as they are completed and leave notes if necessary.
+If an item is not relevant to this project, [strike it out](https://docs.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#styling-text)
+(e.g. `~~Not relevant item~~`) or remove it. If child tickets are created for
+any line-item, please update this ticket description to include references to
```

Comment:
```suggestion
any line-item, please update this description to include references to them.
```

Can we apply this change (or similar) to all other tickets in this PR?

Reply:
✅